auth/auth.js

```diff
@@ -34,7 +33,7 @@ function Auth (options) {
     }
 
     return initializedService.then((service) => {
-      return service.getToken(token).then((result) => {
+      return service.getConsumer(req).then((result) => {
```

Comment:
The req object is quite huge. We should pass only the needed parameters here and not the whole object. Plus the service shouldn't know anything about the req object as it's the responsibility of the controller. Also it's quite cool to name the params with destrucuration like so:

```js
getConsumer({ apiKey: req.get('X-API-Key'), name: req.get('X-User-Name'), scopes: req.get('X-User-Scopes') })
```